packages/frontend/apps/electron/src/main/recording/feature.ts

```
@@ -113,16 +138,106 @@ export const appGroups$ = new BehaviorSubject<AppGroupInfo[]>([]);
 
 export const updateApplicationsPing$ = new Subject<number>();
 
-// There should be only one active recording at a time; state is managed by the state machine
-export const recordingStatus$ = recordingStateMachine.status$;
+export const recordingSessionStatus$ = recordingStateMachine.status$;
```

ESLint

[error] Finnish notation should not be used here.

(rxjs/finnish)